contrib/kubespray/quick-start/services-configuration.yaml.template

```diff
@@ -55,6 +55,37 @@ webportal:
   server-port: 9286
 
 
+#If you want to customize the hivedscheduler vc config or add a new vc,
```

Comment:
#If you want to customize the scheduling config, such add more virtual clusters or more gpu types, check:
#https://github.com/microsoft/pai/blob/master/docs/hivedscheduler/devops.md

Reply:
done